src/System.CommandLine.Hosting/HostingExtensions.cs

```diff
@@ -39,16 +39,15 @@ public static class HostingExtensions
                 hostBuilder.UseInvocationLifetime(invocation);
                 configureHost?.Invoke(hostBuilder);
 
-                using (var host = hostBuilder.Build())
-                {
-                    invocation.BindingContext.AddService(typeof(IHost), _ => host);
+                using var host = hostBuilder.Build();
+
+                invocation.BindingContext.AddService(typeof(IHost), _ => host);
 
-                    await host.StartAsync();
+                await host.StartAsync();
 
-                    await next(invocation);
+                await next(invocation);
 
-                    await host.StopAsync();
-                }
+                await host.StopAsync();
```

Comment:
Might as well follow Visual Studio's `Use simple using statement` (introduced in C# 8) suggestion when updating the package.